pages/_slug/index.vue

```diff
@@ -16,6 +16,11 @@ export default {
 			content: ""
 		}
 	},
+	head () {
+		return {
+			titleTemplate: this.title + " | %s"
```

Comment:
Out of curiosity. How does `%s` know to render as "We Count"

Reply:
https://vue-meta.nuxtjs.org/api/#titletemplate